specification/loadtestservice/resource-manager/Microsoft.LoadTestService/playwright/readme.typescript.md

```diff
@@ -0,0 +1,14 @@
+## TypeScript
+
+These settings apply only when `--typescript` is specified on the command line.
+Please also specify `--typescript-sdks-folder=<path to root folder of your azure-sdk-for-js clone>`.
+
+``` yaml $(typescript)
+typescript:
+  azure-arm: true
+  package-name: "@azure/arm-playwright"
```

Comment:
the previous package name is [@azure/arm-playwrighttesting](https://www.npmjs.com/package/@azure/arm-playwrighttesting)